pci/src/vfio.rs

```diff
@@ -1600,6 +1614,7 @@ impl VfioPciDevice {
                         start: region.start.0 + area.offset,
                         size: area.size,
                         host_addr: host_addr as u64,
+                        dma_map_start: region.start.0 + area.offset,
```

Comment:
What is the difference between `dma_map_start` and `start`?

Reply:
I think this is leftover debug on my part - I will remove dma_map_start and use start instead